src/runtime/button-api.js

```diff
@@ -98,20 +100,17 @@ export class ButtonApi {
 
   /**
    * @param {!Element} button
-   * @param {!Object|function()} optionsOrCallback
-   * @param {function()=} opt_callback
+   * @param {!../api/subscriptions.ButtonOptions|function()} optionsOrCallback
+   * @param {function():null=} opt_callback
    * @return {!Element}
    */
   attach(button, optionsOrCallback, opt_callback) {
-    const options =
-        typeof optionsOrCallback != 'function' ?
-        optionsOrCallback : null;
-    const callback = /** @type {function()} */ (
-        (typeof optionsOrCallback == 'function' ? optionsOrCallback : null) ||
-            opt_callback);
+    const options = this.getOptions_(optionsOrCallback);
+    const callback = /** @type {function(Event):boolean} */
+        (this.getCallback_(optionsOrCallback, opt_callback));
     let theme = options && options['theme'];
```

Comment:
So it looks like now options are never null/undefined here?

Reply:
Yes, updated.

Reply:
Yes. Remove that code updated for `options[lang]` as well.